index.html

```diff
@@ -32,5 +32,127 @@ <h1 class="playfair">Okolo <br> Oshoke</h1>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+        <div class="grid-item-4">            
+            <div class="project-div">
+                <img src="img/project_3.jpg" alt="">
+                <h2 class="poppins">Project name goes here</h2>
+                <ul>
+                    <li class="li-htm poppins">HTML/CSS</li>
+                    <li class="li-rub poppins">Ruby on Rails</li>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+        <div class="grid-item-5">            
+            <div class="project-div">
+                <img src="img/project_4.jpg" alt="">
+                <h2 class="poppins">Project name goes here</h2>
+                <ul>
+                    <li class="li-htm poppins">HTML/CSS</li>
+                    <li class="li-rub poppins">Ruby on Rails</li>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+        <div class="grid-item-6">            
+            <div class="project-div">
+                <img src="img/project_5.jpg" alt="">
+                <h2 class="poppins">Project name goes here</h2>
+                <ul>
+                    <li class="li-htm poppins">HTML/CSS</li>
+                    <li class="li-rub poppins">Ruby on Rails</li>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+        <div class="grid-item-7">            
+            <div class="project-div">
+                <img src="img/project_6.jpg" alt="">
+                <h2 class="poppins">Project name goes here</h2>
+                <ul>
+                    <li class="li-htm poppins">HTML/CSS</li>
+                    <li class="li-rub poppins">Ruby on Rails</li>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+    </section>
+    <!--About Section-->
+    <section class="mobile mobile-about">
+        <div>
+            <h1 class="playfair">About</h1>
+            <p class="poppins">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore 
+                magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+            </p>
+            <button type="button">
+                Get my CV
+                <span><img src="img/ic_download.svg" alt=""></span>
+            </button>
```

Comment:
This button doesn't have the `active` interaction styling. This is where classes would be really helpful. Consider creating a class to apply it to all of your button projects.

<img width="857" alt="Screen Shot 2022-06-29 at 18 06 15" src="https://user-images.githubusercontent.com/82426682/176560261-e42836f1-6cca-4bb4-8599-f28c66dfb4ca.png">